Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'dm-sqlite-adapter'
 gem 'fhir_client'
 gem 'json-jwt'
 gem 'kramdown'
+gem 'minitest'
```

Comment:
Should this be in the `group :test`?

Reply:
I'm pretty sure this is where we get `assert`, so it's used outside of tests.

Reply:
The `assert` in the sequences comes from the `Inferno::Assertions` module if that's what you're referring to.

https://github.com/onc-healthit/inferno/blob/dbb146a7bef896790c6e4695dde6520406740039/lib/app/utils/assertions.rb#L6-L10

Reply:
Well, that seems obvious in retrospect.